src/inmanta/db/versions/v5.py

```diff
@@ -26,5 +26,9 @@ async def update(connection: Connection) -> None:
 -- Compile queue might be collapsed if it contains similar compile requests.
 -- In that case, substitute_compile_id will reference the actually compiled request.
 ALTER TABLE public.compile ADD COLUMN substitute_compile_id uuid REFERENCES public.compile (id);
+
+
+-- Compile data json exported by compiling with the --json parameter.
+ALTER TABLE public.compile ADD COLUMN compile_data json;
```

Comment:
- Originally I did that but I didn't like how it added something compiler-specific to a table which represents a generic report (not necessarily for the `inmanta compile` step of the process). This was my motivation to do it like this, but I'm not fully convinced one way or the other. I've committed my original changes so it should be relatively easy to revert if desired.
- The main reason I opted for json over jsonb was "jsonb will reject numbers that are outside the range of the PostgreSQL numeric data type, while json will not." I thought that with future extensions of the compile data json structure in mind it would be prudent to take the safe approach. But perhaps that shouldn't be taken into consideration at the moment?

Reply:
* Personally I don't see any advantages/disadvantages of doing it one way or another. So to me you can choose how to implement this.
* That's true, but the same also hold for the integer datatype. So I think it's safe to use jsonb.